vendor/assets/javascripts/jquery.textile.js

```diff
@@ -278,9 +280,6 @@
         this.options.$wrap.css('width', 'auto');
         this.options.tmpspan.after(this.options.$wrap).remove();
 
-        this.options.$preview.css('height', '100%');
-        this.$element.css('height', this.options.height);
```

Comment:
No more auto-height?

Reply:
@brianp this didn't do anything because the height is handling in CSS.